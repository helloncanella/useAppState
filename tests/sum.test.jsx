import renderer from "react-test-renderer"
import React from "react"

test("adds 1+2 to equal 3", () => {
  const tree = renderer
    .create(<Link page="https://gogole.co">Google</Link>)
    .toJSON()

  expect(tree).toMatchSnapshot()
  // expect(sum(1, 2)).to.be.equal(3);
})

function Link({ children, page }) {
  return <a href={page}>{children}</a>
}

const testRenderer = renderer.create(
  <Link page="https://google.com">
    Facebook
    <a href="">Hey</a>
  </Link>
)

console.log(testRenderer.toJSON())
