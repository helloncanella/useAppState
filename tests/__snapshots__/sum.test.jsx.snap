// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`adds 1+2 to equal 3 1`] = `
<a
  href="https://gogole.co"
>
  Google
</a>
`;
